app/controllers/attributes_controller.rb

```diff
@@ -0,0 +1,87 @@
+class AttributesController < ApplicationController
+  before_action :set_attribute, only: [:show, :edit, :update, :destroy]
+
+  def index
+    self.params = params.permit!
+    type = Type.find(params[:type_id])
+    @attribute_type_name = type.name
+    @attribute_type_id = type.id
+    @attributes = Attribute.where(type_id: params[:type_id]).order('created_at DESC').page(params[:page]).per(4)
+
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def new
+    @attribute = Attribute.new
+    attribute = Type.find(params[:type_id])
+    @attribute_type_id = attribute.id
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def edit
+    attribute = Type.find(params[:type_id])
+    @attribute_type_id = attribute.id
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create
+    type = Type.find(attribute_params[:type_id])
+    attribute_params[:type_id] = params[:type_id]
+    @attribute = Attribute.new(attribute_params)
+    @attribute.user_id = type.user_id
+    respond_to do |format|
+      if @attribute.save
+        format.js
+        format.html { redirect_to @attribute, notice: 'Attribute was successfully created.' }
+        format.json { render :show, status: :created, location: @attribute }
+      else
+        format.html { render :new }
+        format.json { render json: @attribute.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @attribute.update(attribute_params)
```

Comment:
@basti I thought there was no need for back-end authorization for attributes, because I limited actions on front-end, but know that is not good idea, so I will provide back-end authorization for attributes also.